src/index.html

```diff
@@ -50,6 +54,13 @@ <h1 slot="navbar" class="header-title">Buddhanexus</h1>
         <navigation-menu></navigation-menu>
       </div>
       <main></main>
+      <div style="clear: both;"></div>
     </vaadin-app-layout>
+    <footer>
+        <div class="footerbar">
```

Comment:
naming: please use the `-` sign as a delimiter when defining css classes - e.g `footer-bar`